.gitignore

```diff
@@ -11,3 +11,10 @@
 /yggd.bash
 /yggdrasil.spec
 /doc/tags.toml
+
+.idea
+
+*.pem
+*.rpm
```

Comment:
I don't think this is necessary. I hope to ultimately remove the .copr and spec file completely. I don't like carrying distribution-specific build instructions in the upstream repository.